packages/connector/src/snap.ts

```diff
@@ -266,6 +308,16 @@ export async function createVC(
   this: Masca,
   params: CreateVCRequestParams
 ): Promise<Result<VerifiableCredential>> {
+  if (
+    params.options?.save === true &&
+    (!params.options?.store || params.options?.store.includes('ceramic'))
+  ) {
+    const sessionSet = await verifyAndSetCeramicSession(this);
```

Comment:
In case `verifyAndSetCeramicSession` fails, we should return `ResultObject.error`

Reply:
We return ResultObject.error in `verifyAndSetCeramicSession()` which is then returned in `snap.ts` aswell